force-app/main/default/lwc/hot_isDeceasedBannerRequest/hot_isDeceasedBannerRequest.js

```diff
@@ -0,0 +1,15 @@
+import { LightningElement, wire, api, track } from 'lwc';
+import isDeceased from '@salesforce/apex/HOT_RequestController.isUserDeaceased';
+
+export default class Hot_isDeceasedBannerRequest extends LightningElement {
+    @track isDeceased;
+    @api recordId;
+
+    connectedCallback() {
+        isDeceased({
```

Comment:
Når du legger denne funksjonaliteten inn i eksisterende komponent, holder det om du utvider eksisterende getRecord-metode ved å importere referanse til feltet, og legge det til i fields-lista som argument.
avslutt med en get-metode for å rendre teksten basert på resultatet fra getRecord